hmac-protected/hmac-protected/handler.py

```diff
@@ -0,0 +1,30 @@
+import os, hmac, hashlib
+
+def validateHMAC(message,secret,hash):
+
+    # The hash starts with the encryption method "sha1=" we need to remove it
+    cleanHash = hash[5:]
+    encodedSecret = secret.encode()
+    encodedMessage = message.encode()
+
+    # Encrypt message with secret
+    expectedMac = hmac.new(encodedSecret,encodedMessage,hashlib.sha1)
+    hmacDigest = expectedMac.hexdigest()
+
+    # Compare the received hash with created hash
+    if hmacDigest == cleanHash:
```

Comment:
Why not use `receivedHash` and `createdHash` as the variables that are being compared?  That'd remove the need to explain with a comment.